code/modules/surgery/organs/lungs.dm

```diff
@@ -655,7 +694,7 @@
 	if(breath_pp)
 		var/ratio = safe_breath_min / breath_pp
 		suffocator.adjustOxyLoss(min(5 * ratio, HUMAN_MAX_OXYLOSS))
-		return true_pp * ratio / 6
+		return mole_count * ratio / 6
```

Comment:
spooked about order of operations stuff here, logically it should go `(mole_count * ratio)/6` but parenthesis are like a blanket to me, do you think we should add them here?

Reply:
* and / are commutative, and this isn't a case where floating point shit could butt bite us. So even if it wasn't L->R we'd be ok.
I can do it if you'd like but I think it's fine.

Reply:
if you think it's fine then it's probably alright but i personally just don't like fucking around with this stuff in my code so i thought i should mention this